packages/api/tests/endpoints/test-granules.js

```diff
@@ -1912,37 +1912,20 @@ test.serial('update (PUT) returns bad request when the path param granuleName do
   t.is(body.message, `input :granuleName (${granuleName}) must match body's granuleId (${newGranule.granuleId})`);
 });
 
-test.serial('update (PUT) can set running granule status to queued', async (t) => {
-  const runningGranuleId = t.context.fakeGranules[2].granuleId;
-  const response = await request(app)
-    .put(`/granules/${runningGranuleId}`)
-    .set('Accept', 'application/json')
-    .set('Authorization', `Bearer ${jwtAuthToken}`)
-    .send({
-      granuleId: runningGranuleId,
-      status: 'queued',
-    });
-
-  t.is(response.status, 200);
-  t.deepEqual(JSON.parse(response.text), {
-    message: `Successfully updated granule with Granule Id: ${runningGranuleId}`,
-  });
-});
-
-test.serial('update (PUT) does not set status to queued if not running', async (t) => {
-  const granuleId = t.context.fakeGranules[0].granuleId;
+test.serial('update (PUT) can set granule status to queued', async (t) => {
```

Comment:
I think we also need/want a unit test to prove that the PUT endpoint can create a new granule with a status of "queued", in addition to updating a granule to have a status of queued

Reply:
It's a little clearer to not conflate PUT/update